src/graphql/res_cache.rs

```diff
@@ -0,0 +1,116 @@
+use std::sync::{Arc, RwLock};
+use std::time::Duration;
+
+use async_graphql_value::ConstValue;
+use ttl_cache::TtlCache;
+
+use crate::config::Cache;
+use crate::lambda::{EvaluationContext, Expression, ResolverContextLike};
+
+const DEFAULT_MAX_AGE: u64 = 30;
+
+#[derive(Clone)]
+pub struct ResCache {
+  cache: Cache,
+  data: Arc<RwLock<TtlCache<u64, ConstValue>>>,
```

Comment:
```suggestion
  data: Arc<RwLock<TtlCache<u64, A>>>,
```
Keep data polymorphic.